scripts/cli-migrations/docker-entrypoint.sh

```diff
@@ -11,6 +11,16 @@ log() {
 DEFAULT_MIGRATIONS_DIR="/hasura-migrations"
 TEMP_MIGRATIONS_DIR="/tmp/hasura-migrations"
 
+# configure the target database for migrations
+if [ ${HASURA_GRAPHQL_MIGRATIONS_DATABASE_ENV_VAR} ]; then
```

Comment:
@shahidhk This is my implementation.

I am able to migrate a database, that is not named as `HASURA_GRAPHQL_DATABASE_URL` without needing to have a custom `entrypoint.sh`.

![image](https://user-images.githubusercontent.com/30017294/70806759-ee256100-1db3-11ea-8730-362ce2770c62.png)

Since heroku will always name the first database as `DATABASE_URL`

Reply:
Here is an example of how this would look when using a Heroku manifest (heroku.yml) to manage the application entirely from a committed code base.

https://github.com/simonireilly/graphql-engine-heroku/blob/7b5e9181a759df50f10424226ff5f63a25278b5b/heroku.yml#L1-L9

Reply:
My understanding of the current script is that it will fall back to `HASURA_GRAPHQL_DATABASE_URL` which can't be provisioned at deploy time, so you will need to set up the app, go to the admin panel, and then change the name of the database primary URL.

Reply:
Got it.